unit_tests/sources/declarative/auth/test_oauth.py

```diff
@@ -330,7 +328,7 @@ def test_refresh_access_token_expire_format(
     ):
         next_day = "2020-01-02T00:00:00Z"
         config.update(
-            {"token_expiry_date": pendulum.parse(next_day).subtract(days=2).to_rfc3339_string()}
+            {"token_expiry_date": (ab_datetime_parse(next_day) - timedelta(days=2)).isoformat()}
```

Comment:
Since `__str__` is always generating the compliant format, this should be fine:

```suggestion
            {"token_expiry_date": str(ab_datetime_parse(next_day) - timedelta(days=2))}
```